### ISSUE TITLE
Long delays to send messages

### ISSUE THREAD

**Issue by JCmAxohtwEoK0fdaQip7** [wireapp/wire-desktop#6493]
Wire version: Version 3.30.4368
Wire for web version: Wire for Web Version 2023.03.07.08.22
Operating system: Win7 x64
Which antivirus software do you have installed: None

What steps will reproduce the problem?
1.  Open a chatbox
2.  Write a message
3.  Send the message

What is the expected result?
Message is sent immediately or within a resonable few seconds

What is the actual result?
Message disappears from from entry box, nothing visually happens
Message appears up to a minute later

Basically same as this except the desktop client
https://github.com/wireapp/wire-webapp/issues/14797

**Comment by hamslam** [comment]
A fix for this issue has already been prepared. See https://github.com/wireapp/wire-webapp/pull/14800

**Comment by tlebon** [comment]
This should be fixed. If it appears again please open a new issue.
